packages/common/jest.config.ts

```diff
@@ -1,5 +1,5 @@
 import packageJson from './package.json'
-const packageName = packageJson.name.split('@yisuses/').pop()
+const packageName = packageJson.name.split('@whe/').pop()
```

Comment:
Take a look to comment on packages/admin/jest.config.ts file

Reply:
Done!